nasa/harmony-regression-tests
# Update HGA Sentinel test case 13 for HGA v1.2.6.

## Description

This PR does three things:

* Updates the Sentinel test-case 13 to make it compatible with HGA v1.2.6.
* Removes unnecessary whitespace in a lot of cells.
* Makes sure the disabled tests are disabled via comments, rather than making them a multiline string.

Note - once merged, the updated Sentinel test-case 13 will only pass with HGA v1.2.6 or higher.

## Jira Issue ID

Long-IP, currently no ticket

## Local Test Steps

* Pull this branch.
* Pull the [v.1.2.6 branch of HGA](https://github.com/nasa/harmony-gdal-adapter/pull/23).
* Build that version of HGA with `./bin/build-image`.
* Restart local Harmony instance services with `./bin/restart-services`.
* Run the notebook - all active tests should pass.

## PR Acceptance Checklist
* ~~Acceptance criteria met~~
* ~~Documentation updated (if needed)~~
